### PR TITLE
Adds projection angles load option

### LINKED ISSUE #674: Provide a manual way to input a file that contains projection angles
A `\n` separated file with only a projection angle per line:

```
0.0000
0.1
0.2
...
```

### PR DESCRIPTION
**Pull request by dtasev**
Adds the option to load a projection angles file from the main window.

Also:
- Adds projection angles handling in images. Projection angles from logs are still top priority.
- Adds a simple parser for 2 formats of projection angles

The supported formats are comma separated or new-line separated:
```
0,0.1,0.2,0.3...
```
and
```
0
0.1
0.2
...
```

To test:
- Load a preview stack (only 11 images)
- To test either of the two formats copy and paste this into some file that you can select as a projection angle from File > Load proj angles
	- Test with/without a new line at the end. 

```
0,0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8,0.9,1.0
```
and
```
0
0.1
0.2
0.3
0.4
0.5
0.6
0.7
0.8
0.9
1.0
```
- Try adding/removing angles. It should show an error with the number of each (angles provided and images available)

Fixes #674

### CODE REVIEW

**Review by Pasarus — CHANGES_REQUESTED** [review]
Just a few comments, functionally it works great and is almost ready to be merged!

File: mantidimaging/gui/windows/main/view.py
Comment (dtasev) [reply mantidimaging/gui/windows/main/view.py:183]: We're not expecting any specific files, could be txt, csv, etc. I'd rather just leave it to the users to find it.

File: mantidimaging/core/utility/test/projection_angle_parser_test.py
Comment (dtasev) [reply mantidimaging/core/utility/test/projection_angle_parser_test.py:39]: Good catch, I forgot to remove it after merging it into a single parametrized test

**Review by Pasarus — APPROVED** [review]

**Pull request merged**
